introduction.md

```diff
@@ -1,3 +1,10 @@
 # Helo, my name is Vina and I'm doing Code School Mastering Github course from a nice weather of Singapore. Yoroshiku!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?